yt/yt/tests/integration/master/test_media.py

```diff
@@ -439,9 +500,8 @@ def test_table_medium(self):
             assert replica.attributes["medium"] == self.NON_DEFAULT_MEDIUM
 
     @authors("babenko", "shakurov")
-    @pytest.mark.ignore_in_opensource_ci
```

Comment:
There are a few other places like this, and it is best removed in a separate PR, but I want to get as many tests running in this one as possible, so will do it here for test_media.py.